pkg/vz/vm_darwin.go

```diff
@@ -333,23 +333,11 @@ func attachConsole(_ *driver.BaseDriver, vmConfig *vz.VirtualMachineConfiguratio
 	if err != nil {
```

Comment:
i think we can remove the entire attachConsole method itself
We can re-add this when we provide support for display.

Reply:
Potentially something may not work when no console is connected?

Reply:
It should be fine as far as i know. Not all configs are mandatory / required to be set.

Reply:
removed the entire console